create-pull-request.js

```diff
@@ -11,12 +11,27 @@ const open = require("open");
       throw stderr;
     }
 
+    const browsers = [
+      { id: "chrome", value: "chrome" },
+      { id: "firefox", value: "firefox" },
+      { id: "edge", value: "msedge" },
+      { id: "safari", value: "safari" },
+      { id: "ie", value: "iexplore" },
+      { id: "opera", value: "opera" }
+    ];
+
     const branchName = stdout.trim();
     const remoteOriginUrl = await gitRemoteOriginUrl();
     const newPullRequestUrl = `https://${remoteOriginUrl.replace("git@", "").replace(":", "/").replace(/\.git$/, "")}/pull/new/${branchName}`;
+    let browser = process.argv.pop();
+
+    if (browser) {
+      browser = browsers.find((x) => x.id == browser.toLowerCase());
```

Comment:
We only need the `id` property, so better only pick the id
```suggestion
      browser = browsers.find(({ id }) => id == browser.toLowerCase());
```